### DISCUSSION TITLE
How to load multiple gltf models?

### DISCUSSION THREAD

**Discussion by wddl** [google/filament#7148] [Q&A]
in gltf_viewer example ViewerGui only can set one gltf asset，how can i load and draw multiple gltf asset？thanks

**Comment by romainguy** [comment]
The gltfio library lets you load multiple glTF files, there's nothing special to do. Using an AssetLoader, create multiple FilamentAsset using the createAsset() function.

**Reply by wddl** [reply]
FilamentAsset can create mulitple.but ViewerGui only can set once FilamentAsset.I want set mulitple FilamentAsset  to ViewerGui. wherher need modify ViewGui class or have other method?

**Reply by wddl** [reply]
I got it .ViewerGui is a wrap class .thanks!
